index.js

```diff
@@ -235,13 +234,17 @@ module.exports.cli = function () {
 module.exports.readPackageJson = readPackageJson;
 async function readPackageJson (options, { log } = {}) {
   const opts = options.values();
+  let packageInstance;
   let pkg = {};
   try {
-    pkg = await readPkg(path.resolve(opts.cwd, 'package.json'));
+    packageInstance = await PackageJson.load(opts.cwd, {
+      create: true
+    });
+    pkg = packageInstance.content;
```

Comment:
Not sure if there was a better way, but to reduce the surface of this change I do this so we can just modify the object and leave all those other lines alone.